simulators/main/oeeAPI.py

```diff
@@ -19,7 +20,7 @@ class ProfileCreateMode(Enum):
     CREATE_IF_NOT_EXISTS = 2
 
 class OeeAPI:
-    OEE_BASE = f'{C8Y_BASE}/service/oee-bundle'
+    OEE_BASE = f'{C8Y_BASEURL}/service/oee-bundle'
     CONF_REST_ENDPOINT = f'{OEE_BASE}/configurationmanager/2/configuration'
     SHIFTPLAN_REST_ENDPOINT = f'{OEE_BASE}/mes/shiftplan'
```

Comment:
if we rename "C8Y_BASE" to "C8Y_BASEURL", we should probably also rename "OEE_BASE" to "OEE_BASEURL"